galatasaray-universitesi-sosyal-bilimler-enstitusu.csl

```diff
@@ -0,0 +1,1440 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" delimiter-precedes-et-al="never" page-range-format="chicago" default-locale="tr-TR">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Galatasaray Üniversitesi Sosyal Bilimler Enstitüsü</title>
+    <id>http://www.zotero.org/styles/galatasaray-universitesi-sosyal-bilimler-enstitusu</id>
+    <link href="http://www.zotero.org/styles/galatasaray-universitesi-sosyal-bilimler-enstitusu" rel="self"/>
+    <link href="http://www.zotero.org/styles/chicago-fullnote-bibliography" rel="template"/>
+    <link href="http://dosya.gsu.edu.tr/Sayfalar/2019/2/tez-yazim-kilavuzu-645.doc" xml:lang="tr" rel="documentation"/>
+    <author>
+      <name>Muhammet Tarakçı</name>
+      <email>muhammettarakci@gmail.com</email>
+    </author>
+    <category citation-format="note"/>
+    <category field="social_science"/>
+    <summary xml:lang="tr">Galatasaray Üniversitesi Sosyal Bilimler Enstitüsü'nde yüksek lisans ve doktora tezi yazan öğrencilerimiz için tez yazım kılavuzuna uygun olarak hazırlanmıştır.</summary>
+    <updated>2019-02-20T16:52:22+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="tr">
+    <terms>
+      <term name="editor" form="verb-short">ed.</term>
```

Comment:
A lot of these terms already exist in the locale file (where you provided some of the translations) I'd rather not duplicate them here. Could you please remove all the ones that are identical in the locale?
(Or is this a Mendeley problem again -- in which case this is really up to them to fix. The locale was updated almost a year ago)

Reply:
Dear Adam Smith, 
I have removed some of the local terms. Thank you for your attention.